mobile/README.md

```diff
@@ -0,0 +1,15 @@
+# GnoSocial Mobile APP
+
+## Requirements
+
+- Expo Requiments: https://docs.expo.dev/get-started/installation/
+
+## 🚀 How to use
+
+```sh
+# to run on ios:
+yarn ios 
```

Comment:
When I enter `yarn ios`, it prints "expo: command not found". But if I first enter `yarn`, then `yarn ios` build and runs. Should the README way to first enter `yarn`?

Reply:
Hey @jefft0, I hope the latest commit resolves this issue. Thanks.